docs/reference/templates.rst

```diff
@@ -471,6 +475,38 @@ include your repository in the RenkuLab template source through the
 `renku-values file <https://renku.readthedocs.io/en/latest/admin/index.html#create-a-renku-values-yaml-file>`_.
 
 
+.. _rename-templates:
+
+Renaming templates
+^^^^^^^^^^^^^^^^^^
+
+If you are the owner of a template repository, you can change the name of the
+templates. To do this, edit the ``manifest.yaml`` file and put the new name
+in the ``id`` (``folder``) field of the template you want to rename. You then
```

Comment:
I think we should mention that `folder` is deprecated and should not be used here as well. Because this to me looks like you can use either `folder` or `id`.